### ISSUE TITLE
release: v6.1.0

### ISSUE THREAD

**Issue by Ivy-YinSu** [pingcap/tidb#34048]
### Candidate SHA: 

>  TiDB: 1a89decdb192cbdce6a7b0020d71128bc964d30f
>  TiKV: 080d086832ae5ce2495352dccaf8df5d40f30687
>  PD: d82f4fab6cf37cd1eca9c3574984e12a7ae27c42
>  TiFlash: ebf7ce6d9fb4090011876352fe26b89668cbedc4
>  Tools：
>  binlog: 1012ec778e7bcf03b87d2beccf0f033da2821032
>  tidb-tools: cf6b7a8ae4b5849a6155634352b3712059defb48
>  tiflow: 9f5e3cea2fae0a3dce6cc5ca79e3bcd654b8aa99
>  ng-monitoring: 74826aee5471aecd4e138d56d59261fa9c1ad626
>  enterprise-plugin: c3c5d26eee748dba8c9eaa900e98efc634302c73
>  dumpling: 1a89decdb192cbdce6a7b0020d71128bc964d30f

Working Tags: v6.1.0

### Check the diff in the current version:
[TiDB diff](https://github.com/pingcap/tidb/compare/v6.0.0...v6.1.0)
[TiKV diff](https://github.com/tikv/tikv/compare/v6.0.0...v6.1.0)
[PD diff](https://github.com/tikv/pd/compare/v6.0.0...v6.1.0)
Tools:
[BR diff](https://github.com/pingcap/br/compare/v6.0.0...v6.1.0)
[Binlog diff](https://github.com/pingcap/tidb-binlog/compare/v6.0.0...v6.1.0)
[importer diff](https://github.com/tikv/importer/compare/v6.0.0...v6.1.0)
[tidb-tools diff](https://github.com/pingcap/tidb-tools/compare/v6.0.0...v6.1.0)
[ticdc diff](https://github.com/pingcap/ticdc/compare/v6.0.0...v6.1.0)
[dumpling diff](https://github.com/pingcap/dumpling/compare/v6.0.0...v6.1.0)

-------------------------------
### Release process checklist

- [x] 0. Announce the release plan
- [x] 1. Code freeze: May. 19th
- [x] 2. Build finish: May. 20th
- [x] 3. Qualify the Release. 
- [x] 4. Doc prepared for Release.
 
**Release date: Jun. 13rd, 2022**
Do not proceed below until the release date.

- [x] 5. Publish TiDB Release 
> - [x] fill in Candidate SHA above
> - [x] fill in Make and Publish Build above
- [x] 6. Update version numbers
- [x] 7. Update docs (handled by Docs team)
- [x] 8. Update the offical website

**Comment by Ivy-YinSu** [comment]
Hello Everyone,
We are sorry to announce that TiDB v6.1.0 release will be postponed from 06/09 to 06/13. 

> Because of the following critical issues may cause a quality risk. 
> 
> - https://github.com/pingcap/tiflash/issues/5076
> 
> We have made the adjustments as below PR:
> 
> - https://github.com/pingcap/tiflash/pull/5083
> 
> We treat every quality issue with care, so we will take several days to go through the regression test.

If this change causes you any trouble, please feel free to contact us via [release@pingcap.com](mailto:release@pingcap.com).

Thanks,
Release team
